bom/pom.xml

```diff
@@ -21,15 +21,10 @@
         <version>3.2.2-SNAPSHOT</version>
     </parent>
 
-    <groupId>org.jdbi</groupId>
     <artifactId>jdbi3-bom</artifactId>
     <packaging>pom</packaging>
     <name>jdbi3 BOM</name>
-    <description>Jdbi is designed to provide convenient tabular data access in
-        Java(tm). It uses the Java collections framework for query
-        results, provides a convenient means of externalizing sql
-        statements, and provides named parameter support for any database
-        being used.</description>
+    <description>Jdbi is designed to provide convenient tabular data access in Java(tm). It uses the Java collections framework for query results, provides a convenient means of externalizing sql statements, and provides named parameter support for any database being used.</description>
```

Comment:
is this reformatting helpful?  it's a lot harder to read in diffs now (github is fixed width)

Reply:
I guess not, I just thought the source formatting would mess up formatting in displays elsewhere. Guess I should've asked first

Reply:
reverted